### ISSUE TITLE
Can't do anything with "firebase use --add"

### ISSUE THREAD

**Issue by ahmetb** [firebase/firebase-tools#266]
I get this:

![image](https://cloud.githubusercontent.com/assets/159209/23583795/19d24082-0103-11e7-948f-f76980c67ddd.png)

Then when I hit any arrow keys, the arrow message disappears:

![image](https://cloud.githubusercontent.com/assets/159209/23583798/2717ab60-0103-11e7-9b80-5fe915d883ab.png)

When I hit Return, I get output:

    Error: An unexpected error has occurred.

`debug` file has:

```
[debug] ----------------------------------------------------------------------
[debug] Command:      /usr/local/Cellar/node/7.6.0/bin/node /usr/local/bin/firebase use --add
[debug] CLI Version:  3.4.0
[debug] Platform:     darwin
[debug] Node Version: v7.6.0
[debug] Time:         Sat Mar 04 2017 17:50:38 GMT-0800 (PST)
[debug] ----------------------------------------------------------------------
[debug]
[debug] > command requires scopes: ["email","openid","https://www.googleapis.com/auth/cloudplatformprojects.readonly","https://www.googleapis.com/auth/firebase"]
[debug] >>> HTTP REQUEST GET https://admin.firebase.com/v1/projects
 Sat Mar 04 2017 17:50:38 GMT-0800 (PST)
[debug] <<< HTTP RESPONSE 200
[debug] TypeError: Cannot read property 'short' of undefined
    at Prompt.render (/usr/local/lib/node_modules/firebase-tools/node_modules/inquirer/lib/prompts/list.js:94:69)
    at Prompt.onSubmit (/usr/local/lib/node_modules/firebase-tools/node_modules/inquirer/lib/prompts/list.js:116:8)
    at AnonymousObserver.Rx.AnonymousObserver.AnonymousObserver.next (/usr/local/lib/node_modules/firebase-tools/node_modules/rx-lite/rx.lite.js:1535:12)
    at AnonymousObserver.Rx.internals.AbstractObserver.AbstractObserver.onNext (/usr/local/lib/node_modules/firebase-tools/node_modules/rx-lite/rx.lite.js:1469:31)
    at AnonymousObserver.tryCatcher (/usr/local/lib/node_modules/firebase-tools/node_modules/rx-lite/rx.lite.js:63:31)
    at AutoDetachObserverPrototype.next (/usr/local/lib/node_modules/firebase-tools/node_modules/rx-lite/rx.lite.js:5782:51)
    at AutoDetachObserver.Rx.internals.AbstractObserver.AbstractObserver.onNext (/usr/local/lib/node_modules/firebase-tools/node_modules/rx-lite/rx.lite.js:1469:31)
    at AnonymousObserver._onNext (/usr/local/lib/node_modules/firebase-tools/node_modules/rx-lite/rx.lite.js:4234:13)
    at AnonymousObserver.Rx.AnonymousObserver.AnonymousObserver.next (/usr/local/lib/node_modules/firebase-tools/node_modules/rx-lite/rx.lite.js:1535:12)
    at AnonymousObserver.Rx.internals.AbstractObserver.AbstractObserver.onNext (/usr/local/lib/node_modules/firebase-tools/node_modules/rx-lite/rx.lite.js:1469:31)
[error]
[error] Error: An unexpected error has occurred.
```

I reproduces this very simply by installing firebase-tools via npm, and after running `firebase init` with options: mode=Hosting, "create a new project", ... Then when I run `firebase deploy`, I got:

```
$ firebase deploy
Error: No project active. Run with --project <projectId> or define an alias by
running firebase use --add
```

that's why I used `use --add`.

**Comment by ahmetb** [comment]
Am I the only one who's seeing this? This was very straightforward to repro. In fact, I am holding off on trying out Firebase, because I am currently blocked by this.

**Comment by inlined** [comment]
We will look into why your account isn't able to list associated projects.
While we investigate, you can work around this issue with the --project
option that each command supports.

**Comment by jtrevner** [comment]
I am having the same error. I will use the --project option for now.

**Comment by jtrevner** [comment]
I had the same issue but realized I had created the project with my personal gmail account, rather the my Gsuite account. I authed to firebase tools with my works Gsuite account. Something to double check if anyone comes across this later.

**Comment by mti-karasawa** [comment]
I've got the same problem.
As jtrevner says, I also realized that I was trying 'firebase use --add' with my wrong another account by mistake.
When I tried 'firebase login' with the same account which I used to create a project.

**Comment by guptavj** [comment]
I was using the personal account and it showed projects from there BUT when I switched to the correct account, it still shows projects from person account. So far I have tried these:
1. logout / login: didn't make any difference
2. uninstall firebase-tools and reinstall: didn't help
3. remove ~/.config/configstore/firebase-tools.json: didn't help

How do I get firebase list to show the projects from my current login?

**Comment by mbleigh** [comment]
Is there any chance you have a `FIREBASE_TOKEN` environment variable set?

**Comment by guptavj** [comment]
Yes, that was the problem i found in my investigation.
I was working with Firebase support for few days and let them know yesterday that I had old token in my environment variable.

Had planned to come back here and update the thread.
@mbleigh thank you for your response!!

**Comment by guptavj** [comment]
It would be nice to have the tool report about the invalid token

**Comment by mbleigh** [comment]
Yeah, I think we should at least have a debug log for that.

**Comment by T-light** [comment]
Error: No project active. Run with --project <projectId> or define an alias byrunning firebase use --add

**Comment by T-light** [comment]
please help

**Comment by bkendall** [comment]
@T-light please open a new issue with the appropriate information. Thank you.
